cmake/FindgRPC.cmake

```diff
@@ -0,0 +1,158 @@
+# ~~~
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+#[=======================================================================[.rst:
```

Comment:
I'll assume this syntax means something.

Reply:
Me too :grin:  It is the syntax used by CMake to document their modules.  I cannot say that I understand it, but it is useful as-is (I think) and any mistakes on the markdown can be fixed later.